src/master/filesystem_node_types.h

```diff
@@ -76,6 +76,16 @@ struct statsrecord {
 	uint64_t realsize;
 };
 
+
+[[nodiscard]] [[maybe_unused]]
+static HString hstringToLowerCase(const HString &input) {
+	std::string str = input.c_str();
+
+	std::transform(str.begin(), str.end(), str.begin(), ::tolower);
+
+	return HString(str);
+}
```

Comment:
Because type HString in SaunaFS type of string, consider adding a member method to it, to handle conversions to lower case instead of using global function.

Reply:
Done